AlfrescoKit/AlfrescoKit/Utilities/AKUtility.m

```diff
@@ -0,0 +1,150 @@
+/*
+ ******************************************************************************
+ * Copyright (C) 2005-2015 Alfresco Software Limited.
+ *
+ * This file is part of the Alfresco Mobile SDK.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *****************************************************************************
+ */
+
+#import "AKUtility.h"
+
+static NSDictionary *smallIconMappings;
+static NSString * const kSmallThumbnailImageMappingPlist = @"SmallThumbnailImageMapping";
+
+@implementation AKUtility
+
++ (NSString *)stringDateFromDate:(NSDate *)date
```

Comment:
There's a newer implementation of this function in the app code - the utility class should really be using that one.